tests/test_spectrum_dataframe_helper.py

```diff
@@ -0,0 +1,83 @@
+from typing import List
+
+import pytest
+import pandas as pd
+from matchms.importing import load_from_mgf
+
+from omigami.utilities import SpectrumDataFrameHelper
+
+
+@pytest.fixture
+def spectra_dataframe(small_mgf_path) -> List[pd.DataFrame]:
+    spectra = list(load_from_mgf(small_mgf_path))
+    df = SpectrumDataFrameHelper.from_spectra_list(spectra)
+    return df
+
+
+def test_scale(spectra_dataframe):
+    spectrum_df = spectra_dataframe[0]
+    new_spectrum_df = SpectrumDataFrameHelper.scale(spectrum_df)
+
+    assert spectrum_df.Intensity.max() > 1
+    assert new_spectrum_df.Intensity.max() == 1
+    assert new_spectrum_df.Intensity.min() == 0
```

Comment:
since the min/max are exposed what happens if we pass  to the method `max` < `min`?

Reply:
nice catch, I will raise ValueError and test it in a new test